vaje/tester.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+# TODO: Add documentation comment
+"""BLANK DOCUMENTATION COMMENT"""
+
+__author__ = "Filip Koprivec"
+
```

Comment:
Lahko samo odstraniš to datoteko, ali pa jo premakneš nekam znotraj svoje oddaje?